payments-core/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -197,6 +198,15 @@ sealed interface StripeIntent : StripeModel {
             val hostedVoucherUrl: String? = null,
         ) : NextActionData()
 
+        @Parcelize
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        data class DisplayKonbiniDetails(
+            /**
+             * URL of a webpage containing the voucher for this payment.
+             */
```

Comment:
```suggestion
```

Reply:
In theory we could expose this for bindings. I just copied what the others did.